### DISCUSSION TITLE
发现了一个小问题

### DISCUSSION THREAD

**Discussion by Flyingpen** [Thysrael/Ficus#242] [General]
在榕树模式下生成的思维导图，可能是我内容较多的缘故，导出的PNG格式清晰度太低，无法识别。
正常来说，导出图片应该是矢量图才对。

**Comment by Thysrael** [comment]
这个问题应该会解决的，感谢反馈。

**Comment by GwokHiujin** [comment]
@Flyingpen 您好，导出图片清晰度的问题我们之前已经发现（只是这是一个课程设计项目，在课程第一阶段验收之前没来得及修复），下一阶段确实准备换成 SVG 渲染器，将导出图片格式换成 SVG ，感谢提出。
